saagieapi/apps/apps.py

```diff
@@ -112,7 +118,7 @@ def create_from_scratch(
         check_format_exposed_port = self.check_exposed_ports(exposed_ports)
         if not check_format_exposed_port:
             raise ValueError(
-                "The parameter 'exposed_ports' should be a list of dict. Each dict should contains the key 'port'."
+                "❌ The parameter 'exposed_ports' should be a list of dict. Each dict should contains the key 'port'."
                 "All accept key of each dict is: '{list_exposed_port_field}'"
```

Comment:
Should add fstring for l116, like:
` f"All accept key of each dict is: '{list_exposed_port_field}'"`